src/instructlab/config/config.py

```diff
@@ -1,7 +1,11 @@
-#from lab import ilab
+# from lab import ilab
+# Third Party
 import click
+
+# Local
 from .init import init
```

Comment:
Is this commit mostly changes that the linter/formatter pointed out and you included the fixes here?

Reply:
i moved most (if not all) of the linting into the proper commits